types/vinyl-named/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for vinyl-named 1.1
+// Project: https://github.com/shama/vinyl-named
+// Definitions by: Maiko Tan <https://github.com/MaikoTant>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```

Comment:
```suggestion
```
do you need this explicitely? The setup which uses this module already should have @types/node (say via gulp itself). Adding this explicitely impacts CI tests (ever increasing time).
thanks!

Reply:
I am not very familiar with this. Thanks for pointing that out! 🤣